pkgs/os-specific/linux/anbox/default.nix

```
@@ -85,7 +79,7 @@ stdenv.mkDerivation rec {
     systemd
   ];
 
-  patchPhase = ''
+  prePatch = ''
```

```suggestion
  postPatch = ''
```

otherwise patches no longer rapply.